config-cpp/py_imconfig.h

```diff
@@ -40,3 +40,6 @@ void __py_assert(const char* msg);
 
 // tune vertex index size to enable longer draw lists (see #138 issue)
 #define ImDrawIdx unsigned int
+
+//---- Use 32-bit for ImWchar (default is 16-bit) to support unicode planes 1-16. (e.g. point beyond 0xFFFF like emoticons, dingbats, symbols, shapes, ancient languages, etc...)
+#define IMGUI_USE_WCHAR32
```

Comment:
UTF16 also supports those planes, but it just uses 2 or 3 characters per code point?.

Reply:
I think this comes straight from DearImGui comment documentation of `imconfig.h` (https://github.com/ocornut/imgui/blob/43177324c081868116a99f634e2d5bf38b54056b/imconfig.h#L58), but I must admit I am not at all familiar with character encoding specifications and I just don't know if this is needed or not. Do you suggest that we don't need to apply this change? (this was from pr #241)

Reply:
after re-reading this, the support indicated by the comment is in DearImGui, not in unicode.
DearImGui puts all characters in a dictionary or set or whatever, and that is a whole lot simpler with wchar32's ridiculous big characters.

Reply:
Ok so, if I understand correctly, we need to keep wchar32 in accordance with DearImGui? (i.e. keep this change)

Reply:
indeed.